<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конвертер управляющих программ координатно-пробивного пресса Behrens</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>764</width>
         <height>1084</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="start_conversion">
            <property name="text">
             <string>Начать преобразование</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="conversion_progress">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Каталог, из которого будут забираться программы для преобразования</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="dir_src_layout">
            <item>
             <widget class="QLineEdit" name="dir_src">
              <property name="text">
               <string>.\data</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="dir_src_change">
              <property name="text">
               <string>Выбрать каталог</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Каталог, в котором будут сохраняться преобразованные программы</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="dir_dst_layout">
            <item>
             <widget class="QLineEdit" name="dir_dst">
              <property name="text">
               <string>.\build</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="dir_dst_change">
              <property name="text">
               <string>Выбрать каталог</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Коэфиициент коррекции: F = E \ 100 * &lt;число кодов в минуту&gt; * &lt;коэффициент коррекции&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="coeff_ef_layout">
            <item>
             <widget class="QDoubleSpinBox" name="coeff_ef">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>1</height>
               </size>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>27</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Число ходов в минуту: F = E \ 100 * &lt;число кодов в минуту&gt; * &lt;коэффициент коррекции&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="coeff_e_layout">
            <item>
             <widget class="QDoubleSpinBox" name="coeff_e">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>1</height>
               </size>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
              <property name="value">
               <double>175.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>27</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Cкорость быстрой подачи при перемещении на 10 мм при многократной вырубке в режиме H М25</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="coeff_fast_M25_layout">
            <item>
             <widget class="QDoubleSpinBox" name="coeff_fast_M25">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>1</height>
               </size>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
              <property name="value">
               <double>750.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>27</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <property name="horizontalSpacing">
             <number>1</number>
            </property>
            <property name="verticalSpacing">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="show_m25_warning">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>&lt;p&gt;Вывести все ситуации, когда выполняется пробитие неполных окружностей по М22 и пробиваются дуги радиусом более заданного предела&lt;/p&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Предел радиуса дуги в G02 М22 и G03 M22, после которого будет выводиться предупреждение</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="coeff_G023_M22_layout">
            <item>
             <widget class="QDoubleSpinBox" name="coeff_G023_M22">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>1</height>
               </size>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
              <property name="value">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>27</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="show_verbose">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Вывести подробные сообщения о выполненных преобразованиях</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="show_verbose_m25">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Вывести или нет подробные сообщения о выполненных преобразованиях кадров G02 M25 и G03 M25</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="log_text_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>500</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
